REQUIRE

```diff
@@ -1,4 +1,5 @@
 julia 0.4
+Compat 0.19.0
```

Comment:
I was thinking Interpolations, which I fixed in relation to this fix, since that's a dependency